packages/packages.csv

```diff
@@ -58,9 +59,6 @@ Microsoft.AspNetCore.HttpOverrides,ship
 Microsoft.AspNetCore.Identity,ship
 Microsoft.AspNetCore.Identity.EntityFrameworkCore,ship
 Microsoft.AspNetCore.IISPlatformHandler,noship
-Microsoft.AspNetCore.Internal.libuv-Darwin,noship
-Microsoft.AspNetCore.Internal.libuv-Linux,ship
-Microsoft.AspNetCore.Internal.libuv-Windows,noship
```

Comment:
As long as we still have "libuv-build-windows" in the `package-sources` artifacts it should still be easy to find the `Microsoft.AspNetCore.Internal.libuv-*` packages used to build the `Libuv` package.